src/utils/commonMsg.js

```diff
@@ -51,13 +51,15 @@ export function getActiveTab(type, params) {
       case QUERY_TAB_TYPE.GET_ACTIVE_TAB_BY_URL:
         queryParams = {
           active: true,
-        }
-        if (params && params.origin) {
-          queryParams.url = params.origin + "/*"
+          ...(params && params.origin ? {
+            url: params.origin + "/*",
+          } : {})
```

Comment:
can I say this one's too weird

Reply:
Yeah - it's just the only way of specifying conditional properties so that type can be inferred
```js
        queryParams = {
          active: true,
          ...(params?.origin && {
            url: params.origin + "/*",
          })
        }
```

Reply:
oh this one doesn't even need to be conditional. it works if it's undefined
```js
        queryParams = {
          active: true,
          url: params?.origin ? params.origin + "/*" : undefined,
        }
```